python/src/BetaFactory_doc.i.in

```diff
@@ -21,20 +21,20 @@ Then the distribution bounds are computed from the equations:
     :nowrap:
 
     \begin{eqnarray*}
-      \displaystyle\Hat{a}_n = x_{min} - \frac{\delta}{n + 2}, \\
-      \displaystyle\Hat{b}_n = x_{max} + \frac{\delta}{n + 2}.
+      \hat{a} = x_{(1)} - \frac{\delta}{n + 2}, \\
+      \hat{b} = x_{(n)} + \frac{\delta}{n + 2}.
     \end{eqnarray*}
 
-Let :math:`\bar{x}_n` be the sample mean and :math:`\hat{\sigma}_n` be the sample 
-standard deviation. 
+Let :math:`\bar{x}` be the sample mean and :math:`\hat{\sigma}^2` be the sample 
+unbiased variance. 
 The remaining parameters are estimated from the method of moments:
```

Comment:
I never know if the correct expression is "for" or "by" the method of moments. Maybe "using the method of moments" is best in this instance.

Reply:
Ok. Fixed.